model/src/main/java/com/basistech/rosette/apimodel/Entity.java

```diff
@@ -82,4 +82,9 @@
      */
     private final Label sentiment;
 
+    /**
+     * @return the DBpediaType
+     */
+    private final String dbpediaType;
```

Comment:
Will there be any more KBs? If so, would it be more flexible to have types in a map?

Reply:
The new option in EntitiesOption is missing. I'll add that now.

Reply:
Sounds like we are still deciding what the future value would look like. Simple string might be enough (comma separated list), or just array of strings. Wonder if we want  something like { "default": ["MISC"] }?

Reply:
I was referring to `Map<String, String>`, so that you can have `"extendedEntityTypes": {"dbpedia": "foo", "freebase": "bar", ...}`.

Reply:
After discussing this with Hillel and Hannah, we decided to keep it simple as Beta feature and see if there's a need for a map for multiple kbs (map) or multiple types (array as value) in the future.